mods/cnc/rules/player.yaml

```diff
@@ -59,3 +59,5 @@ Player:
 	ResourceStorageWarning:
 	PlayerExperience:
 	ConditionManager:
+	ProximityCaptor:
```

Comment:
We don't use ProximityCapturable in the default rules for any of the mods, so i'm not sure whether it is worth defaulting this on the player actor? Maps which introduce actors with the trait can just as easily add this too.

This is intended as a discussion point rather than a specific change request.

Reply:
works for me, I can just drop the yaml changes